### PR TITLE
Naukritest changes

### PR DESCRIPTION
**Pull request by veerabhaskar12**
Naukri changes has been updated

### CODE REVIEW

File: src/test/java/com/training/objectmap/NaukriObjectMap.java [hunk 1]
```diff
@@ -6,7 +6,15 @@
 	String NAUKRI_LOGIN_USERNAME_XPATH = "//input[@placeholder='Enter your active Email ID / Username']";
 	String NAUKRI_LOGIN_PASSWORD_XPATH = "//input[@placeholder='Enter your password']";
 	String NAUKRI_CLICK_LOGIN__XPATH = "//button[text()='Login']";
-
```
Comment (chinnaputha) [inline src/test/java/com/training/objectmap/NaukriObjectMap.java:9]: remove unused space.

**Review by chinnaputha — COMMENTED** [review]
review has been done

File: src/test/java/com/training/objectmap/NaukriObjectMap.java (hunk 1 above)
Comment (veerabhaskar12) [reply src/test/java/com/training/objectmap/NaukriObjectMap.java:9]: unused space removed and code committed

**Review by chinnaputha — APPROVED** [review]
Apporving your changes

**Pull request merged**